### PR TITLE
Env cleanup - Remove unused env vars

### PR DESCRIPTION
**Pull request by harimohanraj89**
**Story card:** [ch4166](https://app.clubhouse.io/simpledotorg/story/4166/env-cleanup)

This PR removes crufty env vars from the repo's `.env` files, `app.json`, etc.

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -50,11 +50,9 @@ TWILIO_REMINDERS_ACCOUNT_SID=47912701-5e88-4714-92a0-8f7e473ea43c-fake
 # Feature flags
 #
 ENABLE_AUTO_APPROVE_USER_FOR_QA=false
-ENABLE_DASHBOARD_SNAPSHOT=true
 ENABLE_EXOTEL_WHITELIST_API=false
 ENABLE_FIXED_OTP_ON_REQUEST_FOR_QA=false
 ENABLE_GENERATE_ENCOUNTER_ID_ENDPOINT=true
-ENABLE_PATIENT_FOLLOWUPS=true
```
Comment (harimohanraj89) [inline .env.development:57]: These are unused feature flags

File: .env.development [hunk 2]
```diff
@@ -69,10 +67,6 @@ APPOINTMENT_NOTIFICATION_HOUR_OF_DAY_FINISH=16
 APPOINTMENT_NOTIFICATION_HOUR_OF_DAY_START=14
 APPOINTMENT_NOTIFICATION_FAN_OUT_BATCH_SIZE=
 
-BACKFILL_USER_ID_FROM_AUDIT_LOGS_BATCH_SIZE=1000
-BACKFILL_ENCOUNTERS_FOR_BPS_BATCH_SIZE=1000
-BACKFILL_ENCOUNTERS_FOR_BPS_TIMEZONE_OFFSET=19800
-
```
Comment (harimohanraj89) [inline .env.development:75]: These are unused config vars (remnant of [data migrations](https://github.com/simpledotorg/simple-server/pull/644))

File: .env.development [hunk 3]
```diff
@@ -110,19 +104,12 @@ DEFAULT_COUNTRY=IN
 HELP_SCREEN_YOUTUBE_PASSPORT_URL="https://youtu.be/aktZ1yTdDOA"
 HELP_SCREEN_YOUTUBE_TRAINING_URL="https://youtu.be/MC_45DoRw2g"
 HELP_SCREEN_YOUTUBE_VIDEO_URL="https://youtu.be/nHsQ06tiLzw"
-HELP_SCREEN_SUPPORT_GROUP_URL="http://localhost:3000"
-HELP_SCREEN_SUPPORT_GROUP_NUMBER="+00 0000000000"
-HELP_SCREEN_SUPPORT_GROUP_ICON="icon_whatsapp_icon.svg"
```
Comment (harimohanraj89) [inline .env.development:115]: These are in country-specific .env files now (eg. `.env.IN`)

File: .env.development [hunk 4]
```diff
@@ -110,19 +104,12 @@ DEFAULT_COUNTRY=IN
...
 
 IHCI_ORGANIZATION_UUID=89008920-021e-4116-b74c-8edf86180dd6
 OWNER_EMAILS='Owner <owner@test.com>, Another Owner <another-owner@test.com>'
 PROGRAM_INCEPTION_DATE=2018-01-01
 SUPERVISOR_EMAILS='Supervisor <supervisor@test.com>'
 
-SEED_GENERATED_ADMIN_PASSWORD='Resolve2SaveLives'
-SEED_GENERATED_ACTIVE_USER_ROLE="Seed User | Active"
-SEED_GENERATED_INACTIVE_USER_ROLE="Seed User | Inactive"
-
```
Comment (harimohanraj89) [inline .env.development:125]: `SEED_GENERATED_ADMIN_PASSWORD` is unused. Moved the others to the `.env.seed` files.

**Review by rsanheim — APPROVED** [review]
LGTM

**Pull request merged**